Dockerfile

```diff
@@ -61,4 +67,15 @@ RUN --mount=type=secret,id=composer_auth,dst=/app/auth.json,uid=33,gid=33,requir
     composer install --no-interaction --no-progress --no-dev --optimize-autoloader --classmap-authoritative --no-cache && \
     php artisan nova:publish && \
     php artisan horizon:publish && \
-    sed -i '/HTTPS_ONLY_COOKIES/c\true,' /app/vendor/subfission/cas/src/Subfission/Cas/CasManager.php
+    sed -i '/HTTPS_ONLY_COOKIES/c\true,' /app/vendor/subfission/cas/src/Subfission/Cas/CasManager.php && \
```

Comment:
Why are you modifying vendor package files?

Reply:
That replaces this parameter with a hardcoded `true`: https://github.com/subfission/cas/blob/master/src/Subfission/Cas/CasManager.php#L46

This field cannot be set at runtime because `env` does not work when config is cached.

That should definitely be fixed upstream but I didn't want to deal with it at the time I wrote this.

Reply:
Opened #2475 to track